config/db.js

```diff
@@ -1,7 +1,9 @@
 require('dotenv').config();
 const mongoose = require('mongoose');
+const azureService = require('../utils/azureService');
+
+const db = azureService('MONGO_URL');
```

Comment:
Please, convert `azureService` to actual class and implement method like `getValue` because services are usually classes. Also, please, make this method async because request to the vault is an async action

Reply:
fixed